pulp_rpm/app/models/repository.py

```diff
@@ -148,38 +148,9 @@ class RpmRepository(Repository):
     original_checksum_types = JSONField(default=dict)
     retain_package_versions = models.PositiveIntegerField(default=0)
 
-    def new_version(self, base_version=None):
```

Comment:
strange there was a duplicated thing here ...

Reply:
Line 179 is the only difference between the two

Reply:
Thanks. I don't see where _user_hidden is referenced in either pulpcore or pulp_rpm. How is that aspect maintained by this PR?

Reply:
There's a second PR: https://github.com/pulp/pulpcore/pull/902

Reply:
we removed from pulpcore `plugin_managed` repos concept because it was used only in rpm and as a consequence we have created our own `new_version` in pulp_rpm method because of `sub_repo` https://github.com/pulp/pulpcore/commit/2161bf71ddce76394933fe0d52b09194a2153501#diff-ae59196c41f139ec2b7e3ae3aa581a45

Reply:
Yup, and that's fine, I could also leave it that way... but I was really frustrated that when testing the change using the RPM plugin, I spent several minutes being really confused about why the patch wasn't working before I realized it was going through a separate function.

Reply:
sure, i was not trying to convince doing otherwise, just gave some background history how we ended up having what we have